packages/devtools_app/test/test_infra/scenes/standalone_ui/editor_service/post_message_fake_editor.dart

```diff
@@ -78,6 +78,20 @@ class PostMessageFakeEditor extends EditorServer with FakeEditor {
     connectDevices();
   }
 
+  @override
+  // Always connected for postMessage.
+  bool get connected => true;
+
+  @override
+  Future<void> connectEditor() async {
+    // Not supported for postMessage.
+  }
+
+  @override
+  Future<void> disconnectEditor() async {
+    // Not supported for postMessage.
```

Comment:
I think doing nothing is fine. It just means the Connect/Disconnect buttons in the simulated editor won't do anything for the postMessage version (and not throw), but hopefully this postMessage code isn't going to live long and this whole file will be deleted (I think we can do this much faster than when need to compatible in the other direction, since VS Code extensions tend to be updated fairly quickly).